.github/workflows/release-02_create-draft.yml

```diff
@@ -194,11 +217,12 @@ jobs:
           echo "Found version: >$runtime_ver<"
           echo "::set-output name=runtime_ver::$runtime_ver"
 
-      - name: Fix parachain runtime name
+      - name: Fix runtime name
         id: fix-runtime-path
         run: |
           cd "${{ matrix.runtime }}-runtime/"
           mv "$(sed 's/-parachain/_parachain/' <<< ${{ matrix.runtime }})_runtime.compact.compressed.wasm" "${{ matrix.runtime }}_runtime.compact.compressed.wasm" || true
+          mv "$(sed 's/-kusama/_kusama/' <<< ${{ matrix.runtime }})_runtime.compact.compressed.wasm" "${{ matrix.runtime }}_runtime.compact.compressed.wasm" || true
```

Comment:
I like the idea, let me test it first but yes, that sounds better.

Reply:
While I prefer the generic aspect and the reduction your suggestion brings, I prefer not to that change, at least not for now, too to avoid excessive **greediness** on the regexp.

Today we are tring to change `foo-bar` into `foo_bar` but we would not like to change `foo-baz-bar` into `foo_baz_bar` (it should be `foo-baz_bar`) or neither `foo-bar-baz` into `foo_bar_baz` (it should be `foo_bar-baz`).